Api/Auth/Policies.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace LearnNet_CatalogService.Api.Auth
+{
+    public static class Policies
```

Comment:
Please add  'catalog' prefix to policies to require scopes specific to catalogService